stix2/sources/__init__.py

```diff
@@ -58,7 +59,10 @@ def get(self, *args, **kwargs):
                 object specified by the "id".
 
         """
-        return self.source.get(*args, **kwargs)
+        try:
+            return self.source.get(*args, **kwargs)
+        except AttributeError:
+            raise AttributeError('%s has no data source to query' % self.__class__.__name__)
```

Comment:
Are the original error messages not clear enough? I agree these are a bit more explicit, but the default `'NoneType' object has no attribute 'get'` isn't too bad, I don't think, when pointing to a line that includes `self.source.get(...)`

Reply:
I think the intent with these error messages was to help the user get to the cause of the issue more quickly. For this PR, I just noticed that while we were checking them on Environment, we weren't on DataStore, and I wanted to be consistent.

This also handles the case where a user creates an Environment without specifying a DataStore or DataSource. It will automatically create a CompositeDataSource with no sources, so if the user tries to do `environ.get()` they'd get an error message about CompositeDataSource rather than Environment.

Reply:
Makes sense